src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_acl_constraints.erl

```diff
@@ -0,0 +1,25 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Mark  Mzyk <mm@chef.io>
+%% Copyright 2015 Chef, Inc. All Rights Reserved.
+
+-module(oc_chef_authz_acl_constraints).
+
+-export([check_acl_constraints/4]).
+
+check_acl_constraints(AuthzId, Type, AclPerm, Ace) ->
+  check_admin_group_removal_from_grant_ace(AuthzId, Type, AclPerm, Ace).
```

Comment:
If we use true|false instead of ok, what would you think of the following:

```erlang
acl_checks() ->
    [
     check_admin_group_removal_from_grant_ace/4
    ].

check_acl_constraints(AuthzId, Type, AclPerm, Ace) ->
    lists:all(apply_check_fun(AuthzId, Type, AclPerm, Ace), 
              mandatory_checks()).

apply_check(AuthzId, Type, AclPerm, Ace) ->
    fun(Check) -> 
            apply(Check, [AuthzId, Type, AclPerm, Ace])
    end.
```

which might make it easier to add new checks in the future.